client/src/styles/PortfolioFilters.scss

```diff
@@ -370,17 +370,63 @@
       &[open] {
         --filter-bar-top-height: 4.6rem;
         --filter-bar-height: 6.6rem;
-        position: absolute;
+        position: fixed;
         top: 0;
         left: 0;
         right: 0;
+        z-index: 101;
```

Comment:
lol is this z-index 101 because the vertical gradient at the right side of the table is z-index 100???

Reply:
lol yea, not sure exactly how things got to 100, but yeah otherwise it blurs through. I'll make a ticket to reset all of these z-indexes to something more logical

Reply:
from my quick reading into z-indices it seems like anything that isn't 0 or -1 is discouraged because of how it explicitly sets the tab order, so 100 / 101 aren't bad values to me personally! was just wondering for the giggles

Reply:
oh interesting, I hadn't thought about the effect on tab order, that's good to know. yeah I was about to go deep into z-index and "stacking context" for something in here but found a better solution so have saved that for later.